src/mixins/_button.scss

```diff
@@ -109,11 +109,9 @@
   }
 
   /**
-   * Toggling button
-   * This adds a slash across the icon to visually show the state has changed
-   * when `aria-pressed` is `'true'`.
+   * Adds a slash across the "extra" content (most times "extra" content is an icon).
    */
-  &[aria-pressed='true'] {
+  &.c-button--slashed {
```

Comment:
Should this be a state class instead, like `.is-slashed`?

Reply:
Great question, @tylersticka!

I actually started with a state class. Then wondered if it should be a modifier. Now, you've got me wondering again. LOL 😅

Reply:
It really depends on whether or not we'd ever want this property to be something we could toggle with JavaScript.

If we consider this an inert/static stylistic difference, then a modifier is appropriate.

If we consider this a potentially temporary state the icon is in (like on or off), then a state class would be more appropriate.

I'm fine either way, I just wanted to poke at this a bit before we moved forward. I'd love another opinion from @cloudfour/dev

Reply:
I appreciate you elaborating on your thoughts, @tylersticka!

I could very well see this CSS class being toggled with JavaScript. I'm rolling with a CSS state class. Thank you for pushing back on this! 😄

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1519/commits/3d6fbe3abff61720a5f7af96a6c70d35f843a747